commands.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs ? import <nixpkgs> {}
+, prefix ? "glm"
+# this default "fetch" is for nix-build but we will pass here the golem-examples
+# from flake to make it easier to update
+, golem-examples ? pkgs.fetchFromGitHub { 
+            owner = "golemcloud";
+            repo = "golem-examples";
+            rev = "5165dcc7e3cbfa09f752caa96a869d284ec169aa";
```

Comment:
Why this revision? What happens when we update `golem-exampels`? (we do it quite frequently)